GETTING STARTED - Developers.rdoc

```diff
@@ -0,0 +1,61 @@
+==Is this doc for you?
+
```

Comment:
Agreed, that it shouldn't be README, although README_DEVELOPERS(.ext) might be good.  Or something like ORGANIZATION or SOURCE_OVERVIEW maybe?